config/config.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+return [
+    'bookings_front_ui' => [
+        'holder_template' => '<div data-eddbk-wizard-holder><eddbk-wizard :config=\'%s\'></eddbk-wizard></div>',
+        'edd_settings' => get_option('edd_settings'),
+
+        /*
+         * Configuration for placeholder templates.
+         */
+        'templates_config' => [
+            'token_start'   => '${',
+            'token_end'     => '}',
+            'token_default' => '',
+        ],
+
+        'application_selector' => '[data-eddbk-wizard-holder]',
+        'api_base_urls' => [

```

Comment:
Looks like endpoints or as OpenAPI calls them "paths", rather than base URLs - which are something else, and typically would contain the domain/IP address.

Reply:
Fixed 👌